server/helpers.go

```diff
@@ -61,7 +61,7 @@ func (p *Plugin) PopulateMeeting(m *dataStructs.MeetingRoom, details []string, d
 	m.AttendeePW_ = "ap"
 	m.ModeratorPW_ = "mp"
 	m.Record = "true"
```

Comment:
Why don't you set this to the config value, too?

Reply:
@moschlar great question. As per [the docs](https://docs.bigbluebutton.org/dev/api.html#create), the `record` flag requires manually starting the recording to work. It won't matter setting its value from config. Do you think it has an effect?

Reply:
I thought that when `record=true`, the BBB server always records everything, and in post-processing, the recording will be processed and published or deleted based on the existence of start stop markers.
So your change would basically always make BBB record and discard it afterwards, which would be a waste of resources that would be mitigated by `record=false`.

Reply:
Hmm, that makes sense. I'll update for the same.

Reply:
Done. This is now available in [v4.0.2](https://github.com/blindsidenetworks/mattermost-plugin-bigbluebutton/releases/tag/v4.0.2)